config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  # root "users#index"
+  get "/", to: "users#index"
```

Comment:
We talked about this but maybe in a refactor we can boil everything down in to just resources but this works for now and helps me understand what's going where.